src/main/webapp/app/detail-overview-list/detail-overview-list.component.ts

```diff
@@ -92,15 +83,6 @@ export class DetailOverviewListComponent implements OnInit, OnDestroy {
         }, {});
     }
 
-    showGitDiff(gitDiff?: ProgrammingExerciseGitDiffReport) {
```

Comment:
Can you update the tests so they no longer try to call this method on the `DetailOverviewListComponent`? I think the build is failing